packages/pds/src/api/com/atproto/admin/resolveModerationReports.ts

```diff
@@ -1,15 +1,15 @@
 import { CID } from 'multiformats/cid'
 import { AtUri } from '@atproto/uri'
 import { InvalidRequestError } from '@atproto/xrpc-server'
-import { Server } from '../../../lexicon'
-import AppContext from '../../../context'
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
 import {
   ACKNOWLEDGE,
   FLAG,
   TAKEDOWN,
-} from '../../../lexicon/types/com/atproto/admin/moderationAction'
-import { ModerationAction } from '../../../db/tables/moderation'
-import { InputSchema as ActionInput } from '../../../lexicon/types/com/atproto/admin/takeModerationAction'
+} from '../../../../lexicon/types/com/atproto/admin/moderationAction'
+import { ModerationAction } from '../../../../db/tables/moderation'
+import { InputSchema as ActionInput } from '../../../../lexicon/types/com/atproto/admin/takeModerationAction'
 
 export default function (server: Server, ctx: AppContext) {
   server.com.atproto.admin.takeModerationAction({
```

Comment:
think this one & resolveModerationReport got mixed up files

Reply:
Oops, thanks for catching that!